src/components/coin-icon/CoinIconFallback.js

```diff
@@ -77,6 +79,9 @@ const CoinIconFallback = fallbackProps => {
     address,
   ]);
 
+  const eth = isETH(address);
```

Comment:
Could you also wrap isEth with useMemo or the function itself with memoFn

Reply:
I can wrap this, but this just a string comparison so I am not sure is this is worth it

Reply:
Yeah, you're right, I was looking at another function implementation isEthAddress.